requirements.txt

```diff
@@ -0,0 +1,3 @@
+sphinx==7.4.7
+sphinx-autodoc-typehints==2.3.0
+furo==2024.8.6
```

Comment:
Oh my mistake. I'd suggest adding a requirements.txt for the project - it was late and I didn't investigate that all too much and just put it in root. I'll fix this though and generate a requirements.txt for the actual project if you'd like.

Reply:
Thanks! I appreciate it, once you make those changes I can merge! 

I never really thought about having the `project.toml` and `requirements.txt`. I did a quick search and it is recommended to include both. So sure you can add that in as well if you want!

Reply:
Ok, I moved the docs requirements.txt back and put a requirements.txt with just requests (that's it?) in root. Sorry about that.

I also formatted the main .py with black - it changed some indenting depth, but otherwise mostly the same.

Readded the exit to session close, (once again sorry about being out of date and not rebasing)

I think I covered everything?

Reply:
Look's good!